frontend/src/assets/hcloud.svg

```diff
@@ -0,0 +1,81 @@
+<svg id="Ebene_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 141.7 141.7" data-darkreader-mode="filter"><head xmlns=""><style id="dark-reader-style">@media screen {
+
+/* Leading rule */
+html {
+  -webkit-filter: invert(100%) hue-rotate(180deg) brightness(85%) contrast(90%) !important;
+  filter: invert(100%) hue-rotate(180deg) brightness(85%) contrast(90%) !important;
+}
+
+/* Reverse rule */
+img,
+video,
+:not(object):not(body)&gt;embed,
+object,
+svg image,
+[style*="background:url"],
+[style*="background-image:url"],
+[style*="background: url"],
+[style*="background-image: url"],
+[background],
+twitterwidget,
+.sr-reader,
+.sr-backdrop {
+  -webkit-filter: invert(100%) hue-rotate(180deg) !important;
+  filter: invert(100%) hue-rotate(180deg) !important;
+}
+[style*="background:url"] *,
+[style*="background-image:url"] *,
+[style*="background: url"] *,
+[style*="background-image: url"] *,
+input,
+[background] *,
+img[src^="https://s0.wp.com/latex.php"],
```

Comment:
for what is this external page used? Can this be removed? Could you also double check the other css rules etc if they are really required?

Reply:
Let's use the svg from [here](view-source:https://www.hetzner.com/assets/Uploads/icon-circle-cloud.svg)